ci/crossbeam-epoch.sh

```diff
@@ -13,13 +13,15 @@ if [[ "$RUST_VERSION" == "nightly" ]]; then
     cargo check --no-default-features --features nightly
     cargo test --features nightly
 
-    ASAN_OPTIONS="detect_odr_violation=0 detect_leaks=0" \
-    RUSTFLAGS="-Z sanitizer=address" \
-    cargo run \
-        --release \
-        --target x86_64-unknown-linux-gnu \
-        --features sanitize,nightly \
-        --example sanitize
+    if [[ "$OS" != "windows" ]]; then
```

Comment:
how about saying "if the os is linux"?  Because we're targetting on "x86_64-unknown-linux-gnu" in this code.

Reply:
Sounds good. 

Also, I changed to get `$OSTYPE` directly from bash instead of from the environment variable set in CI settings.